ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFAvatarGroupStyle: Int, CaseIterable {
+    case stack
+    case pile
+}
+
+@objc public protocol MSFAvatarGroupState {
+    var maxDisplayedAvatars: UInt32 { get set }
```

Comment:
Any particular reason you decided to use UInt32 for maxDisplayedAvatars and UInt for overflowCount?

Reply:
I wanted to maintain UInt for both but could not because when I iterate through the avatars using ForEach, the max Uint value goes out of range when cast to Int. Uint32 was a way to address that -- I could also make overflowCount UInt32?

Reply:
It's worth noting that the swift language guide recommends using Int anywhere possible, even for values known to be nonnegative

> NOTE
>
> Use UInt only when you specifically need an unsigned integer type with the same size as the platform’s native word size. If this isn’t the case, Int is preferred, even when the values to be stored are known to be nonnegative. A consistent use of Int for integer values aids code interoperability, avoids the need to convert between different number types, and matches integer type inference, as described in Type Safety and Type Inference.

https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html#ID317